docs/scripts/api/buildApi.ts

```diff
@@ -53,10 +53,14 @@ const createProject = (options: CreateProgramOptions): Project => {
 };
 
 async function run() {
-  const outputDirectories = FEATURE_TOGGLE.enable_product_scope
-    ? ['./docs/pages/api-docs/data-grid', './docs/pages/x/api/data-grid']
-    : ['./docs/pages/api-docs/data-grid'];
-  await Promise.all(
+  let outputDirectories = ['./docs/pages/api-docs/data-grid'];
+  if (FEATURE_TOGGLE.enable_product_scope) {
+    outputDirectories = ['./docs/pages/api-docs/data-grid', './docs/pages/x/api/data-grid'];
+  }
+  if (FEATURE_TOGGLE.enable_redirects) {
+    outputDirectories = ['./docs/pages/x/api/data-grid'];
+  }
+  await Promise.allSettled(
```

Comment:
```suggestion
  await Promise.all(
```
It has to throw if there's an issue, see https://github.com/mui-org/material-ui-x/pull/3808

Reply:
I don't get why it says it's outdated :D
I can't see this change in the code

Reply:
me too 🥲. will do it manually, thanks a lot!